.github/workflows/presubmit.yml

```diff
@@ -217,10 +217,10 @@ jobs:
           clang --version
           cd D:\a\taichi\taichi
           python -m pip install --user numpy pybind11 gitpython
-          $env:CI_SETUP_CMAKE_ARGS = "-G'Visual Studio 16 2019' -A x64 -DPYTHON_EXECUTABLE='$env:PYTHON' -DLLVM_DIR='C:\taichi_llvm\lib\cmake\llvm'"
           python misc/ci_setup.py ci
         env:
           PYTHON: C:\hostedtoolcache\windows\Python\3.7.9\x64\python.exe
+          CI_SETUP_CMAKE_ARGS: -A x64 -DLLVM_DIR='C:\taichi_llvm\lib\cmake\llvm'
```

Comment:
Did you drop `-G'Visual Studio 16 2019'` intentionally?

Reply:
Correct, this is done by default with the msbuild path added. Also, passing this with spaces leads to confusion when parsing cmake_args.